add PROJECT.md file (#132)
openjs-foundation/bootstrap: PROJECTS.md

# OpenJS Foundation Projects

> The projects below are full OpenJS Foundation members at the stage level indicated in the heading. Projects may move to new stages upon annual review or by request. 

Reviewer: Optional simplification: Split long-ish sentence into two short and direct sentences:

```suggestion
The projects below are full OpenJS Foundation members. They are at the stage level indicated in the headings. Projects may move to new stages upon annual review or by request. 
```

> For more information on the stage definitions and how projects move between stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.

Reviewer: Optional text simplification:

```suggestion
For information on the stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.
```

## Impact Projects

* [Appium](http://appium.io/)
* [Dojo](https://dojo.io/)
* [jQuery](https://jquery.com/)

> * [Node.js](https://nodejs.dev/)

Reviewer: The link should be https://nodejs.org/, no?

Reviewer: ```suggestion
* [Node.js](https://nodejs.org/)
```

Author: Whatever ya'll prefer! I wondered if you'd like .dev site because its new and shiny? No matter to me!

Reviewer: The .dev site is a tutorial. The .org site is where you go to download Node.js, read the reference documentation, etc. I would definitely prefer the .org site.

* [webpack](https://webpack.js.org/)

## Growth Projects

* [Architect](https://arc.codes/)
* [Intern](https://theintern.io/)
* [MochaJS](https://mochajs.org/)
* [Node-RED](https://nodered.org/)
* [WebdriverIO](https://webdriver.io/)
* [webhint](https://webhint.io/)

## At Large Projects

* [ESLint](https://eslint.org/)
* [Esprima](http://esprima.org/)

> * [Express](https://expressjs.com/)

Reviewer 2: Still trying to get feedback from Doug on this front.

Author: per conversation in meeting today, moving Express to 'TBD'
